build-info/docker/image-info.dotnet-dotnet-docker-nightly.json

```diff
@@ -1,2575 +1,2172 @@
+              "simpleTags": [
+                "3.1-focal",
+                "3.1.2-focal"
+              ],
+              "digest": "sha256:0c5b0805c226034a52e22cee3dbe92e890c3beb844538151ca93ec24c3d02830",
+              "osType": "Linux",
+              "osVersion": "Ubuntu 20.04",
+              "architecture": "amd64",
+              "created": "2020-03-23T13:22:36.5633712Z"
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "3.1/aspnet/focal/arm64v8/Dockerfile",
+              "simpleTags": [
+                "3.1-focal-arm64v8",
+                "3.1.2-focal-arm64v8"
+              ],
+              "digest": "sha256:632a720bf43fdb63db8058d36b1851c77be59e926cecde5aef7ecef8220d2db1",
+              "osType": "Linux",
+              "osVersion": "Ubuntu 20.04",
+              "architecture": "arm64",
+              "created": "2020-03-23T13:25:48.8224612Z"
+            }
+          ]
+        },
+        {
+          "manifest": {
+            "digest": "sha256:5f53bb47d2146fad10e58278490b2198278674a77cb08106feaf8f8753ead189",
+            "created": "2020-03-23T13:24:44.6081702Z",
+            "sharedTags": [
+              "5.0.0-preview",
+              "5.0",
+              "latest"
+            ]
+          },
+          "platforms": [
+            {
+              "dockerfile": "5.0/aspnet/buster-slim/amd64/Dockerfile",
+              "simpleTags": [
+                "5.0-buster-slim",
+                "5.0.0-preview-buster-slim"
+              ],
+              "digest": "sha256:3dda56a68bba56509d59142e2577e1d327961a5689d3d6a4f59b6a85e9cbb409",
+              "baseImageDigest": "buildpack-deps@sha256:51994825fd939c0c7b6d3d1b3ea01be7411bd5c534ea2694139a4dd2ef00d64d",
+              "osType": "Linux",
+              "osVersion": "Debian 10",
+              "architecture": "amd64",
+              "created": "2020-03-23T13:22:38.0001466Z"
+            },
+            {
+              "dockerfile": "5.0/aspnet/buster-slim/arm32v7/Dockerfile",
+              "simpleTags": [
+                "5.0-buster-slim-arm32v7",
+                "5.0.0-preview-buster-slim-arm32v7"
+              ],
+              "digest": "sha256:123158895ecc74d856f6e8083f725551bf0b840ab9aec31fce350213c28e1a59",
+              "baseImageDigest": "arm32v7/buildpack-deps@sha256:c342b21207b0428d0b5542d6640518cb1b13d88867f1bf6c1846eb133eb6e328",
+              "osType": "Linux",
+              "osVersion": "Debian 10",
+              "architecture": "arm32",
+              "created": "2020-03-23T13:30:24.5166081Z"
+            },
+            {
+              "dockerfile": "5.0/aspnet/buster-slim/arm64v8/Dockerfile",
+              "simpleTags": [
+                "5.0-buster-slim-arm64v8",
+                "5.0.0-preview-buster-slim-arm64v8"
+              ],
+              "digest": "sha256:7ef29c87cdd67a7b13388f12d85d5b047fe03644783cf14cc603f791e0133ecf",
```

Comment:
Is it inconsistent that the image digest is just the digest where as the baseImageDigest is repo qualified?  The image is scoped to the containing repo which defines the scope.  I think the answer is does it help the consumer to qualify the image digest?

Reply:
That's a fair question.  From a human readability standpoint, I can see how it would be useful to have it fully qualified because a lot of scrolling could be required to find the repo name.

Reply:
I've logged https://github.com/dotnet/docker-tools/issues/448 to track this.